evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -20,10 +20,15 @@
                     <tr class="vote-row d-flex">
                         {% if field|is_choice_field %}
                             <td class="vote-question vote-question-choice my-auto{% if field.errors %} choice-error{% endif %}" name="{{ field.name }}">
+                            {{ field.label }}
                         {% else %}
-                            <td class="vote-question vote-question-text">
+                            <td class="vote-question vote-question-text d-flex flex-column">
+                            {{ field.label }}
+                            <div class="warning-label mt-auto d-none">
+                                <span class="fas fa-exclamation-triangle"></span>
+                                {% trans "This answer will be filtered out. Please leave the textbox empty if you don't have any feedback for this particular question." %}
```

Comment:
This message does not clearly state that a human will process the answer. It could also mean that it's automatically filtered. Do we want this?

Reply:
What about "This answer seems obsolete. Please leave [...]"? I was using what @janno42 suggested at #1197.

Reply:
"This answer seems obsolete" sounds good. I would also not put the human processing into focus here.

Reply:
obsolete means outdated and i didn't find any other meanings that would match...?

personally i'm fine with the current solution in this PR, though i would prefer my shorter proposal `This comment will be filtered out. You can simply leave the textbox empty.` I think that's nice than "Please leave this empty" which imo is a little strong, and without the human work aspect it's not clear why we would care that much.

I think the mere appearance of some text that suggests a different behavior will be enough for most users to change behavior (of course not generally but in this particular case, since i'd guess that users have no particular desire to enter something in there).

Reply:
Yes, obsolete is the wrong word here. I personally don't like "This comment will be filtered out" because i would say that this sounds too strong. So what about "You can simply leave the textbox empty if you don't have any feedback for this question."?

Reply:
sorry, i realized i commented on an outdated diff and deleted the comment to re-post it below...